src/import-map-injector.ts

```diff
@@ -112,3 +116,5 @@ function injectImportMap(importMaps: ImportMap[]): void {
   finalImportMapScriptEl.textContent = JSON.stringify(finalImportMap);
   document.head.appendChild(finalImportMapScriptEl);
 }
+
+export {};
```

Comment:
Needed in order to be able to import the module in Jest. Otherwise, it complains the file is not a module.